packages/bsky/src/api/app/bsky/feed/getFeed.ts

```diff
@@ -38,14 +38,19 @@ export default function (server: Server, ctx: AppContext) {
       const db = ctx.db.getReplica()
       const feedService = ctx.services.feed(db)
       const viewer = auth.credentials.iss
-
+      const headers: Record<string, string> = {}
+      for (const [key, value] of Object.entries(req.headers)) {
+        if (typeof value === 'string') {
+          headers[key] = value
+        }
+      }
```

Comment:
I think there's enough complexity with the hop-by-hop headers that we may just want to special-case `accept-language` for now until we can do a more rigorous pass on this.

@CooperEdmunds are there any other headers that you'd like to have passed through to the feed generator?

Reply:
accept-language is sufficient. I'll make that change.

Reply:
yeah, my intuition would be to pass-through only an allow-list of headers, not all-by-default.

In addition to divy's list above, there is `Host`, and security/privacy sensitive headers like `X-Real-IP`, `X-Forwarded-Proto`, `Strict-Transport-Security`, `Cookie`, etc.

Other ones I could see wanting to pass through are:

- `Accept` (content-negotiation)
- `Cache-Control` (eg, if trying to cache-bust?)
- `If-Modified-Since` (cache control)
- `Max-Forwards` (prevent loops!)
- `Origin` (for CORS? and logging? possible privacy concern?)
- `Referer` (possible privacy concern?)
- `User-Agent`
- `DNT` (do-not-track)